api/pom.xml

```diff
@@ -45,6 +44,27 @@
     <project.directory>api</project.directory>
   </properties>
 
+  <build>
+    <pluginManagement>
+      <plugins>
+        <plugin>
+          <groupId>org.codehaus.modello</groupId>
+          <artifactId>modello-maven-plugin</artifactId>
+          <executions>
+            <execution>
+              <id>modello-site-docs</id>
+              <goals>
+                <goal>xdoc</goal>
+                <goal>xsd</goal>
+              </goals>
+              <phase>none</phase>
```

Comment:
This works?

Reply:
yes: it "disables" the execution that is enabled by default, ie. bound to real existing lifecycles

Reply:
Nice...

Reply:
> Nice...

Not really.  It's a workaround for the inability to disable a plugin execution imho.  But it works...